src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -234,7 +234,7 @@ bool CryptoManagerImpl::OnCertificateUpdated(const std::string& data) {
 }
 
 SSLContext* CryptoManagerImpl::CreateSSLContext() {
-  if (context_ == NULL) {
+  if ((context_ == NULL) || (IsCertificateUpdateRequired())) {
```

Comment:
@tkireva Please apply the yoda style and remove the brackets, simpler is better, isn't it?